war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -212,6 +212,15 @@ var createPluginSetupWizard = function(appendTarget) {
 						'<button type="button" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></button>');
 				}
 
+				// add Jenkins version
+				if(translations.installWizard_jenkinsVersionTitle) { // wait until translations available
```

Comment:
@kzantow 

> No, it will fall back to the English translations. 

So, in what situation will this not be displayed? Am I missing something here?